geopmdpy/test/TestActiveSessions.py

```diff
@@ -202,7 +202,7 @@ def check_json_file(self, name, contents, is_valid):
             mock_srf.assert_called_once_with(full_file_path)
 
             if is_valid:
-                mock_get_session_path.called_once_with(contents['client_pid'])
```

Comment:
A tech debt issue or similar should be filed about why this code path wasn't being hit.